src/main/java/org/jenkinsci/plugins/awsdevicefarm/AWSDeviceFarmRecorder.java

```diff
@@ -329,9 +329,9 @@ public AWSDeviceFarmRecorder(String projectName,
 
         if (ifWebApp != null && ifWebApp) this.ifWebApp = true;
```

Comment:
This line doesn't make sense. If it's not null and true, set it to true. The conditional won't be fulfilled if it isn't already truthy.

Use `boolean` (lowercase) in the constructor if you're somehow guarding against some edge case?